x-pack/plugins/search_indices/server/lib/status.ts

```diff
@@ -62,7 +62,8 @@ export async function fetchUserStartPrivileges(
     logger.error(e);
     return {
       privileges: {
-        canCreateIndex: false,
+        canManageIndex: false,
```

Comment:
Seems like some sort of error should be passed along rather than silently restricting access